cfgov/unprocessed/apps/paying-for-college/css/college-costs/college-costs.less

```diff
@@ -167,6 +179,12 @@
     	list-style: none;
 	}
 
-
-	
+    .o-table__financial {
+    	.indented-row td:first-child {
+        	padding-left: unit( @grid_gutter-width * 2/3 / @base-font-size-px, em );
```

Comment:
`padding-left: unit( @grid_gutter-width / 1.5 / @base-font-size-px, em );` might be clearer?